LeaderboardBackend/Models/ViewModels/RunViewModel.cs

```diff
@@ -0,0 +1,34 @@
+using LeaderboardBackend.Models.Entities;
+using NodaTime;
+
+namespace LeaderboardBackend.Models.ViewModels;
+
+public record RunViewModel
```

Comment:
Pending discussion in Discord, but if we do keep `Status` in the parent model, we should return it here, too.

Reply:
There is no run approval mechanism as of right now. Newly created runs appear instantly. Runs can be deleted so there needs to be a mechanism for that, even if it's just marking them with a deleted flag. I would think the api does not return deleted runs by default. Maybe a query parameter can be used for that.

Reply:
In other words, the `Status` field should not exist at this time. I recommend `IsDeleted` or `Deleted` for handling the deletion mechanism.

Reply:
@Dalet yeah let's add an `IsDeleted` then I'll resolve this.

Reply:
^ to be done in a future PR.